### PR TITLE
#67853 check if tvos tests are still failing

### PR DESCRIPTION
**Pull request by mkhamoyan**

### CODE REVIEW

File: src/libraries/System.IO.Pipes/src/System/IO/Pipes/PipeStream.Unix.cs [hunk 1]
```diff
@@ -30,7 +30,7 @@ public abstract partial class PipeStream : Stream
         private static readonly char[] s_invalidPathNameChars = Path.GetInvalidPathChars();
 
         /// <summary>Prefix to prepend to all pipe names.</summary>
-        private static readonly string s_pipePrefix = Path.Combine(Path.GetTempPath(), "CoreFxPipe_");
+        private static readonly string s_pipePrefix = Path.Combine(Path.GetTempPath(), (RuntimeInformation.IsOSPlatform(OSPlatform.Create("TVOS"))) ? "" : "CoreFxPipe_");
```
Comment (adamsitnik) [inline src/libraries/System.IO.Pipes/src/System/IO/Pipes/PipeStream.Unix.cs:33]: A few months back I was working on reducing shared memory object name for macOS arm64: https://github.com/dotnet/runtime/pull/68541

I am wondering whether you could be facing `Interop.Error.ENAMETOOLONG` here as well? What error are you getting on tvOS?

Comment (mkhamoyan) [reply src/libraries/System.IO.Pipes/src/System/IO/Pipes/PipeStream.Unix.cs:33]: I was getting this exception 
Exception messages: System.ArgumentOutOfRangeException : The path '/private/var/mobile/Containers/Data/Application/46CE2A72-8096-4780-861F-BA91283A36A6/tmp/CoreFxPipe_jfmkhnmo' is of an invalid length for use with domain sockets on this platform.  The length must be between 1 and 104 characters, inclusive. (Parameter 'path')
But removing "CoreFxPipe_" prefix didn't help , as after that I started getting 
Exception messages: System.Net.Sockets.SocketException : Permission denied   Exception stack traces:    at System.Net.Sockets.Socket.UpdateStatusAfterSocketErrorAndThrowException(SocketError error, Boolean disconnectOnFailure, String callerName)

**Pull request closed without merging**